docs/tutorials/prep_hyp3_for_mintpy.ipynb

```diff
@@ -29,20 +29,35 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import os\n",
     "import re\n",
-    "import glob\n",
     "import zipfile\n",
     "import numpy as np\n",
     "import hyp3_sdk\n",
     "from pathlib import Path\n",
-    "from osgeo import gdal\n",
-    "proj_dir = os.path.expanduser('/media/jzhu4/data/hyp3-mintpy/test_prep_ts_hyp3')\n",
-    "proj_name = os.path.basename(proj_dir)\n",
-    "hyp3_dir = os.path.join(proj_dir, 'hyp3')\n",
-    "mintpy_dir = os.path.join(proj_dir, 'mintpy')\n",
-    "os.makedirs(hyp3_dir, exist_ok=True)\n",
-    "os.makedirs(mintpy_dir, exist_ok=True)"
+    "from osgeo import gdal"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Define needed paths and create analysis directories"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "proj_name = 'test_prep_ts_hyp3'\n",
+    "proj_dir = Path.cwd().joinpath(proj_name)\n",
+    "hyp3_dir = proj_dir.joinpath('hyp3')\n",
+    "dirs = [proj_dir, hyp3_dir]\n",
+    "\n",
+    "for dir in dirs:\n",
+    "    if not dir.is_dir():\n",
+    "        dir.mkdir()"
```

Comment:
```suggestion
    "proj_name = 'test_prep_ts_hyp3'\n",
    "proj_dir = Path.cwd() / proj_name\n",
    "proj_dir.mkdir(exist_ok=True)\n",
    "\n"
    "hyp3_dir = proj_dir /'hyp3'\n",
    "hyp3_dir.mkdir(exist_ok=True)"
```

Generally, I'd prefer to use the `/` opperator instead of `joinpath` with `pathlib`, and not to loop for just two directories. 

You can also use `mkdir(exist_ok=True)` instead of `if not dir.is)dir(): dir.mkdir()`.